__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>work on your variable names.</p><p> </p><p>When i see a variable called gradientboost_models i think it should be a list of models. Instead it contains accuracies. Guess what I would suggest as a better variable name? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I should have used index and score in the name of the variable instead.</p><p> However, I intentionally wanted to create a list for the scores and indices </p>